CHANGELOG.md

```diff
@@ -213,6 +213,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 * [CHANGE] Reduce period of health checks in connection pools for querier->store-gateway, ruler->ruler, and alertmanager->alertmanager clients to 10s. This reduces the time to fail a gRPC call when the remote stops responding. #3168
 * [CHANGE] Hide TSDB block ranges period config from doc and mark it experimental. #3518
 * [FEATURE] Alertmanager: added Discord support. #3309
+* [ENHANCEMENT] Go: update go go 1.20.1. 
```

Comment:
This is in the wrong place, it needs to be under "main / unreleased".

Reply:
[nit] Also there's a little typo. Should be "update go to...".